code/game/objects/machinery/overwatch.dm

```diff
@@ -1214,7 +1244,129 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	dat += "<A href='?src=\ref[src];operation=hide_dead'>{[dead_hidden ? "Show Dead Marines" : "Hide Dead Marines" ]}</a><br>"
 	dat += "<A href='?src=\ref[src];operation=choose_z'>{Change Locations Ignored}</a><br>"
 	dat += "<br><A href='?src=\ref[src];operation=back'>{Back}</a>"
-	return dat
+	return dat			
+
+/obj/machinery/computer/camera_advanced/overwatch/send_orders(datum/source, atom/object)
+	var/turf/target_turf = get_turf(object)
+	if (current_order)
+		if(COOLDOWN_CHECK(src, cooldown_order_cic))
+			COOLDOWN_START(src, cooldown_order_cic, ORDER_COOLDOWN)
+			switch (current_order)
+				if (ATTACK_ORDER)
+					send_attack_orders(target_turf)
+					return
+				if (DEFEND_ORDER)
+					send_defend_orders(target_turf)
+					return
+				if (RETREAT_ORDER)
+					send_retreat_orders(target_turf)
+					return
+		to_chat(usr, "<span class='warning'>Your last order was too recent</span>")
+		return
+	return
```

Comment:
```suggestion
```
no need for return in end of the proc

Reply:
Tivi likes useless return, but i'll remove this one